src/basic/fileio.h

```diff
@@ -26,6 +26,7 @@ typedef enum {
         WRITE_STRING_FILE_NOFOLLOW              = 1 << 8,
         WRITE_STRING_FILE_MKDIR_0755            = 1 << 9,
         WRITE_STRING_FILE_MODE_0600             = 1 << 10,
+        WRITE_STRING_VIRTUALFILE_SUPRESS_IF_POSSIBLE       = 1 << 11,
```

Comment:
format

Reply:
maybe name this WRITE_STRING_SUPPRESS_REDUNDANT_VIRTUAL?